test/main_dialog_spec.rb

```diff
@@ -257,4 +257,29 @@ def expect_update_table_with(expected_items)
       end
     end
   end
+
+  describe "#help_handler" do
+    before do
+      mock_ui_events(:help, :cancel)
+    end
+
+    it "launch a help popup" do
+      expect(Yast::Popup).to receive(:LongText).with(
```

Comment:
too fragile, any change in help text break this test. I suggest to only verify that it will receive some text?